biochatter/rag_agent.py

```diff
@@ -203,9 +203,9 @@ def generate_responses(self, user_question: str) -> list[tuple]:
         ]:
             final_answer = self.query_func(user_question)
             if final_answer is not None:
-                response = [("response", final_answer)]
+                response = [(final_answer, "response")]
```

Comment:
Here we aligned return values of API with those of KG agent and Database agent: (content, metadata)